tests/browser-polyfill.ts

```diff
@@ -1,7 +1,15 @@
-export const browser = {
+import { BackgroundRequest } from '../src/background/background-request';
+import { calculateEraDateTimeSpan } from '../src/background/calculate-date';
+
+const browser = {
   runtime: {
     getURL: () => '#',
-    sendMessage: () => new Promise((resolve) => resolve),
+    sendMessage: (request: BackgroundRequest) => {
+      if (request.type === 'calculateEraDateTimeSpan') {
+        return Promise.resolve(calculateEraDateTimeSpan(request));
+      }
+      return new Promise((resolve) => resolve);
```

Comment:
This is my mistake but I guess this could just be:

```suggestion
      return Promise.resolve();
```